CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+We are a team with limited capacity dedicated to several projects. Every single contribution is valid and more than welcome. Feel free to help us in developing new ABAP Checks and keeping this project up & running! Hereby a short how-to tutorial.
+
+### How-to
+
+Fork our repository, then clone/pull the forked repo via ABAPgit into your system:
+
+    https://github.com/your-username/code-pal-for-abap.git
```

Comment:
I would say here that the user has to choose the `dev` branch (by default, it came as master).